pkg/validation/strfmt/default.go

```diff
@@ -154,13 +156,13 @@ func init() {
 	Default.Add("hostname", &hn, IsHostname)
 
 	ip4 := IPv4("")
-	Default.Add("ipv4", &ip4, govalidator.IsIPv4)
+	Default.Add("ipv4", &ip4, isIPv4)
```

Comment:
You have one replacement here `IsIPv4String`
https://github.com/kubernetes/utils/blob/efc7438f0176a4a29c62bac8989d8dd183054fa5/net/net.go#L145

Reply:
mm... the implementation isn't quite the same... I'm not confident it would handle `::1.2.3.4` addresses the same way

Reply:
then better to be on the safe side, and not change anything, I thought it was simpler but is not